lib/config/config-file.js

```diff
@@ -492,15 +493,14 @@ function resolve(filePath, relativeTo) {
  */
 function load(filePath, applyEnvironments, relativeTo) {
     const resolvedPath = resolve(filePath, relativeTo),
-        dirname = path.dirname(resolvedPath.filePath),
-        lookupPath = getLookupPath(dirname);
```

Comment:
@AsaAyers This `lookupPath` will become different by the installed place. This is the start point of module resolving.